apps/dashboard/config/configuration_singleton.rb

```diff
@@ -372,6 +372,10 @@ def open_apps_in_new_window?
     end
   end
 
+  def api_enabled?
+    to_bool(ENV['OOD_API_ENABLED'])
+  end
```

Comment:
I think we should limit this to batch connect api.  Again, with the files api (which already exists and maybe with no way to turn it off?) we can at least turn one off and keep the other.

Also it seems like this needs a default `|| false`?

Reply:
Not sure I understand.
Is it that you want a more granular control to enable/disable each of the endpoints? like:
```
  def api_enabled?(api_name)
    to_bool(config.dig(:api_config, api_name.to_s))
  end
```
So that I can configure:
```
api_config:
 sessions_api: true
 applications_api: false
 launchers_api: false
```

Or just to change this one in particular to:
```
 def sessions_api_enabled?
   to_bool(ENV['OOD_SESSIONS_API_ENABLED'])
 end
```

PS: `to_bool` will return false when the value is `nil`, so it will default to false in this case.

Reply:
Yea I think this is what I'm looking for. It's tough with a config, once you settle on it and tell folks about it you can't really change it and you have to pray it's granular enough to only impact the one thing it's supposed to do.

```
 def batch_connect_api_enabled?
   to_bool(ENV['OOD_BATCH_CONNECT_API_ENABLED'])
 end
```

Reply:
Changes implemented